escrow/state_service.proto

```diff
@@ -24,9 +24,11 @@ service PaymentChannelStateService {
 message ChannelStateRequest {
     // channel_id contains id of the channel which state is requested.
     bytes channel_id = 1;
+    //current block number (signature will be valid only for short time around this block number)
```

Comment:
This will break backward compatibility. The new field should be assigned a new number